www/app/Http/Middleware/Authenticate.php

```
@@ -0,0 +1,44 @@
+use Illuminate\Http\Request;
+
+/**
+ * Bypass-aware authentication middleware.
+ *
+ * Extends Laravel's built-in Authenticate middleware to honour PocketDev's
+ * two auth-bypass modes (permanent and session-based).  Without this,
+ * routes that apply the `auth` middleware as defense-in-depth would reject
+ * bypassed users — they pass through EnsureSetupComplete (which knows about
+ * bypass) but fail the standard Auth::check() because no user is logged in.
+ */
+class Authenticate extends BaseAuthenticate
```

PHPStan (2.1.54)

[error] Class App\Http\Middleware\Authenticate extends unknown class Illuminate\Auth\Middleware\Authenticate.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)